tests/qa_automation/qa_run.pm

```diff
@@ -50,7 +50,12 @@ sub system_status {
 sub system_login {
     my $self = shift;
     $self->wait_boot;
-    select_console('root-console');
+    if (get_var('VIRTIO_CONSOLE')) {
+        select_console('root-virtio-terminal');
+    }
+    else {
+        select_console('root-console');
+    }
```

Comment:
@okurz where ? in this test are only two places where is selected console, and second isn't needed :) Plus for this test is virtio_console good performance optimisation, unfortunately this breaks SLE12GA/SLE12SP1 usage and needs wait for pr#2820